pandas/tests/io/formats/test_to_latex.py

```diff
@@ -448,6 +448,169 @@ def test_to_latex_longtable(self, frame):
         with3columns_result = df.to_latex(index=False, longtable=True)
         assert r"\multicolumn{3}" in with3columns_result
 
+    def test_to_latex_caption_label(self, frame):
+        the_caption = 'a table in a \\texttt{table/tabular} environment'
+        the_label = 'tab:table_tabular'
+
+        frame.to_latex(
+            caption=the_caption,
+            label=the_label
+        )
```

Comment:
What is this expression testing?

Reply:
I put this here to be consistent with the first line of code in the existing `test_to_latex()`, `test_to_latex_format()`, and `test_to_latex_longtable()`.  Is this a convention of some sort?

Reply:
@gfyoung it appears the pytest fixture `frame` is not required in the following methods contained under the `master` branch:

- `TestToLatex.test_to_latex()`
- `TestToLatex.test_to_latex_format()`
- `TestToLatex.test_to_latex_longtable()`
- `TestToLatex.test_to_decimal()`

Should this be corrected under this PR or should a new issue be created?

Reply:
Save for a separate PR.

Reply:
> Is this a convention of some sort?

Uncertain, but it does look strange to me.